types/react-native/index.d.ts

```diff
@@ -1659,7 +1659,7 @@ export interface ViewPropertiesAndroid {
 
 type Falsy = undefined | null | false
 interface RecursiveArray<T> extends Array<T | RecursiveArray<T>> {}
-interface RegisteredStyle<T> extends Number {}
+export interface RegisteredStyle<T> extends Number {}
```

Comment:
Because `StyleSheet.create` just return a ID, see https://facebook.github.io/react-native/docs/stylesheet.html

And this is previous exist code, i havn't do any change.